modules/libcom/src/error/errlog.c

```diff
@@ -719,3 +719,16 @@ static void errlogThread(void)
     if(wakeFlusher)
         epicsEventMustTrigger(pvt.waitForSeq);
 }
+
+int * arrayFunction(int size, size_t size_t) {
+    static int * array;
+
+    //array = calloc(size, sizeof(int));
+    //array = malloc(size * sizeof(int));
+    array = malloc(size * size_t);
+
+    for (int i=0; i< size;i++){
+        array[i] = i;
+    }
+    return array;
+}
```

Comment:
github reports "no newline at end of tile"

Reply:
I am using vscode on a linux machine, but I will look into this now

Reply:
You should be able to force vscode to insert newlines by adding this file at the top of epics-base: https://github.com/epics-base/epics-base/blob/01b6ea9c1d35f2cba80fc7a3ef30da5cccd4ff6a/.editorconfig

Reply:
Thanks for pointing this out; I have changed my IDE settings so it automatically inserts a newline at the end of any edited files. I have since deleted this function, which I really just used to aid my understanding, but hopefully this should not be a problem again.